adobe/spectrum-web-components
# Fix Table a11y test failing

## Description

Added a tab stop to the TableBody upon rendering the Virtualised Table so that the a11y test no longer flakes. 

## Motivation and context


Moving it closer to going into main :) 

## How has this been tested?
Visually, and with the a11y test passing.



    1. Go to storybook
    2. Tab into the page
    3. See that you can't tab into the tableBody
    1. Go to Tachometer Storybook
    2. Tab into the page
    3. See that you CAN, in fact, tab into the tableBody! 

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
